app/page.tsx

```diff
@@ -45,6 +47,13 @@ export default async function Index() {
           />
         </div>
       </div>
+
+      <div className="mt-20 flex flex-col items-center gap-2 text-center">
```

Comment:
Can we just remove mt here as we already have a gap on the wrapper?

<img width="658" alt="Screenshot 2024-12-20 at 15 08 39" src="https://github.com/user-attachments/assets/f786021f-dba3-48ce-9e0c-fa990e91e622" />
 

```suggestion
      <div className="flex flex-col items-center gap-2 text-center">
```